.drone-v1.yml

```diff
@@ -0,0 +1,258 @@
+---
+kind: pipeline
+type: kubernetes
+name: build
+
+trigger:
+  event:
+    - push
+
+steps:
+  - name: install dependencies
+    image: node:14.15.4-alpine
+    commands:
+      - npm --loglevel warn install --production=false --no-optional
```

Comment:
Seems odd that we force dev dependencies into our image? But that's for another time.

Reply:
We need the dependencies for the build and test - unless I'm misunderstanding